f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -56,7 +56,9 @@ def create_listener(self, service, bigips):
             tls['name'] = vip['name']
             tls['partition'] = vip['partition']
 
+        network_id = service['loadbalancer']['network_id']
         for bigip in bigips:
+            self.service_adapter.get_vlan(vip, bigip, network_id)
```

Comment:
I am not sure about that.  Since the members' responses are routed back to the BigIP via the SNAT address, it isn't necessary, but I can't speak to why it would be desirable or not.  It might be a question that should be referred to John.  It should be easy to extend the functionality to add member networks, but my thought would be that the code could be implemented in the pool service.

Rich

Sent from my iPad

> On Jun 22, 2016, at 5:55 PM, Matt Greene <notifications@github.com> wrote:
> 
> In f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py:
> 
> >          for bigip in bigips:
> > +            self.service_adapter.get_vlan(vip, bigip, network_id)
> Let me ask the question differently. The VS is the aggregation point for listeners and members. This PR locks down the VLAN or tunnel added the VS to restrict traffic on the listener side. But do we also need to add VLAN or tunnel to the VS for members that live on other networks than the listeners?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>